src/bp/ui-studio/src/web/reducers/flows.ts

```diff
@@ -237,7 +239,7 @@ const doCreateNewFlow = name => {
     }
   ]
 
-  const isSubWorkflow = window.USE_ONEFLOW && parseFlowName(name).folders?.length
+  const isSubWorkflow = false // TODO window.USE_ONEFLOW && parseFlowName(name)
```

Comment:
A reusable workflow need outputs: success/failure node

Reply:
In the UI implementation, reusable workflows are stored in __reusable__ folder so that's what gets checked here